### ISSUE TITLE
[Accessibility]: Repo Management - Can't navigate to "Connect a code host"

### ISSUE THREAD

**Issue by jplahn** [sourcegraph/sourcegraph#36417]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33519

### Problem description

You can't navigate to the "Connect a code host" link on the https://sourcegraph.com/users/[username]/settings/repositories page using the tab key 

### Expected behavior

I should be able to open the "Your repositories" page and navigate to "Connect a code host" with the tab key 

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by subercui** [comment]
https://sourcegraph.com/users/xxx/settings/repositories/manage

Hi, I can't find a way to connect to GitHub and add repos. The above link encounters 404 and there is just no `repositories` tab in the sidebar of the settings webpage. 

![image](https://user-images.githubusercontent.com/11674033/177014150-39a3c502-59f1-4155-924c-b60f266ca95e.png)


I guess this is related to this issue? Is there a workaround so far?

**Comment by subercui** [comment]
Yeah, I'm using my personal account to add private repos. However, I can't see the "host connections" tab in the sidebar navigation.

Following this link https://docs.sourcegraph.com/code_search/how-to/adding_repositories_to_cloud

**Comment by jplahn** [comment]
Thanks for the info @subercui 

We recently decided to sunset user-added repositories on [Sourcegraph.com](http://sourcegraph.com/), which is why new repositories can’t be added at the moment. [Sourcegraph.com](http://sourcegraph.com/) will still support searching our open source code index, and if you’d like a way to search private repositories for free, we recommend setting up a [local install of Sourcegraph](https://docs.sourcegraph.com/#quick-install).

You can also read more about the changes to [Sourcegraph.com](http://sourcegraph.com/) in our [blog](https://about.sourcegraph.com/blog/single-tenant-cloud).

**Comment by subercui** [comment]
Thanks a lot @jplahn . Sorry, I didn't notice that.
